File: BossFight Rest/BossFight/boss_fight_py_edition/DamageTrackerEntry.py
class DamageTrackerEntry:
    def __init__(self, player_id, player_name, duration, damage):
        self._player_id = player_id
        self._player_name = player_name
        self._duration = duration
        self._damage = damage

    def to_dict(self) -> dict:
        return {"player_id": self._player_id, "player_name": self._player_name, "duration": self._duration, "damage": self._damage}

    def subtract_turn(self, n: int = 1):
        self._duration -= n

    def get_duration(self):
        return self._duration

    def get_player_id(self):
        return self._player_id

    def get_player_name(self):
        return self._player_name

    def get_damage(self):
        return self._damage

File: BossFightBackEnd/Models/DamageTrackerEntry.cs
using System.Collections.Generic;

namespace BossFight.Models
{
    public class DamageTrackerEntry
    {
        private int _player_id { get; set; }
        private string _player_name { get; set; }
        private int _duration { get; set; }
        private int _damage { get; set; }

        public DamageTrackerEntry(int player_id, string player_name, int duration, int damage)
        {
            _player_id = player_id;
            _player_name = player_name;
            _duration = duration;
            _damage = damage;
        }

        public void SubtractTurn(int n = 1)
        {
            _duration -= n;
        }

        public int GetDuration()
        {
            return this._duration;
        }

        public int GetPlayerId()
        {
            return this._player_id;
        }

        public string GetPlayerName()
        {
            return this._player_name;
        }

        public int GetDamage()
        {
            return this._damage;
        }
    }
}